src/pages/booking/BookingPage.vue

```diff
@@ -35,46 +35,88 @@
-          <q-btn
-            v-for="(adviser, index) in 20"
-            :key="index"
-            @click="
-              TheDialogs.emit({
-                type: 'studentBookingDialog',
-                arg: {
-                  success() {
-                    console.log('IM STUDENT');
-                  },
-                },
-              })
-            "
-            class="bg-primary q-pa-md"
-            :class="['q-md:w-18', 'q-sm:w-45', 'q-xs:w-100']"
-          >
-            <q-icon name="person" color="white" />
-            <span class="text-bold text-white">Adviser {{ index + 1 }}</span>
-          </q-btn>
-        </div>
+      <!-- <q-table flat bordered grid :rows="advisers" :columns="columns" /> -->
+      <div class="flex-wrap row items-start q-gutter-md">
+        <q-card
+          class="adviser-card col-auto"
+          v-for="adviser in advisers"
+          :key="adviser.key"
+        >
+          <q-card-section class="text-h6">{{
+            adviser.fullName
+          }}</q-card-section>
+          <q-separator />
+          <q-card-section>
+            <span v-for="column in columns" :key="column.name">
+              {{ column.label }}: {{ adviser[column.name] }} <br
+            /></span>
+          </q-card-section>
+          <q-card-actions align="right">
+            <q-btn
+              flat
+              icon="insert_invitation"
+              label="SET APPOINTMENT"
+              @click="openBookingDialog"
+            />
+          </q-card-actions>
+        </q-card>
       </div>
     </div>
   </q-page>
 </template>
 
 <script setup lang="ts">
 import { TheDialogs } from 'src/dialogs/the-dialogs';
-
-import { ref } from 'vue';
+import { QTableColumn } from 'quasar';
+import { IAdviserProfile } from 'src/entities';
+import { useProfileStore } from 'src/stores/profile.store';
+import { computed, onMounted, onUnmounted, ref } from 'vue';
 
 const isSearchExpanded = ref(false); // Tracks if the search button is expanded
 const searchQuery = ref<string>(''); // Stores the input value
+const profileStore = useProfileStore();
+let sub: ReturnType<typeof profileStore.streamProfiles> | undefined;
+
+const advisers = computed(() => {
+  return profileStore.profiles;
+});
+
+const columns = [
+  // {
+  //   name: 'fullName',
+  //   label: 'Name',
+  //   align: 'left',
+  //   field: (row: IAdviserProfile) => row.fullName,
+  // },
+  {
+    name: 'college',
+    label: 'College',
+    align: 'left',
+    field: (row: IAdviserProfile) => row.college,
+  },
+  {
+    name: 'department',
+    label: 'Department',
+    align: 'left',
+    field: (row: IAdviserProfile) => row.department,
+  },
+  {
+    name: 'position',
+    label: 'Position',
+    align: 'left',
+    field: (row: IAdviserProfile) => row.position,
+  },
+] as QTableColumn[];
+
+onMounted(() => {
+  sub?.unsubscribe();
+  sub = profileStore.streamProfiles({ type: 'adviser' });
```

Comment:
You may want to use findProfiles instead of streamProfiles.
**Reasons:**

1. Profiles does not update frequently
2. streamProfiles is heavy on server side.
3. Booking is better for streaming because anytime can be created, updated, or deleted where as Profiles is created once and don't usually update often.